src/main/java/rotodo/commands/DeleteCommand.java

```diff
@@ -0,0 +1,30 @@
+package rotodo.commands;
+
+import rotodo.exception.InvalidInputException;
+import rotodo.processes.Storage;
+import rotodo.processes.Ui;
+import rotodo.tasklist.TaskList;
+
+/**
+ * The DeleteCommand class encapsulates the specific
+ * type of Command that executes a delete action.
+ *
+ * @author Ng Kay Hian
+ * @version CS2103T AY24/25 Semester 1
+ */
+public class DeleteCommand extends Command {
+    private int idx;
```

Comment:
Perhaps you could consider using the full name spelt out instead of "texting style" spelling like we learnt this week. Just a nitpick though.